### DISCUSSION TITLE
Don't follow redirects

### DISCUSSION THREAD

**Discussion by cumminsd** [thunderclient/thunder-client-support#779] [Q&A]
I've been using Postman to test the login and API authorization chain of calls. When Okta calls back with a redirect, the authorization code is a query parameter on the URL in the redirect call. Postman allows you to turn off automatically following the redirects and then has the ability to pull the authorization code off the URL and store it in an env variable. 

Is it possible to do the same in Thunder Client?

**Comment by rangav** [comment]
Hi @cumminsd , we have vscode setting to turn off redirects
